File: lib/confirmation_result.dart
class ConfirmationResult {
  String confirmationIntent;
  String confirmedResult;
  String voiceInput;
  bool isSuccess;

  ConfirmationResult(
      {this.confirmationIntent,
      this.confirmedResult,
      this.voiceInput,
      this.isSuccess});

  ConfirmationResult.fromJson(Map<String, dynamic> json) {
    confirmationIntent = json['confirmationIntent'];
    confirmedResult = json['confirmedResult'];
    voiceInput = json['voiceInput'];
    isSuccess = json['isSuccess'];
  }

  Map<String, dynamic> toJson() {
    final Map<String, dynamic> data = new Map<String, dynamic>();
    data['confirmationIntent'] = this.confirmationIntent;
    data['confirmedResult'] = this.confirmedResult;
    data['voiceInput'] = this.voiceInput;
    data['isSuccess'] = this.isSuccess;
    return data;
  }

  @override
  String toString() {
    return 'ConfirmationResult{confirmationIntent: $confirmationIntent, confirmedResult: $confirmedResult, voiceInput: $voiceInput, isSuccess: $isSuccess}';
  }
}

File: android/src/main/kotlin/com/umair/background_stt/models/ConfirmationResult.kt
package com.umair.background_stt.models

data class ConfirmationResult(var confirmationIntent: String = "", var confirmedResult: String = "", var voiceInput: String, var isSuccess: Boolean) {

    override fun toString(): String {
        return "{" +
                "  \"confirmationIntent\": \"$confirmationIntent\"," +
                "  \"confirmedResult\": \"$confirmedResult\"," +
                "  \"voiceInput\": \"$voiceInput\"," +
                "  \"isSuccess\": $isSuccess" +
                "}"
    }
}